src/Build/Evaluation/IItemTypeDefinition.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable disable
```

Comment:
nit:
Do you need to disable nullable here?

Reply:
Removed